vscode/src/edit/input/get-matching-context.ts

```diff
@@ -11,17 +15,25 @@ interface FixupMatchingContext {
     key: string
     /* If present, will override the key shown in the quick pick selector */
     shortLabel?: string
-    file: ContextItem
+    item: ContextItem
 }
 
 export async function getMatchingContext(
     mentionQuery: MentionQuery
 ): Promise<FixupMatchingContext[] | null> {
+    if (mentionQuery.type === 'empty') {
```

Comment:
Maybe can use `getChatContextItemsForMention` here so that every time we add a new context item (like the URL context we just added a few days ago) we won't need to add it again for Edit separately?  Something like:

```suggestion
export async function getMatchingContext(
    mentionQuery: MentionQuery
): Promise<FixupMatchingContext[] | null> {
    const token = new vscode.CancellationTokenSource()?.token
    const results = await getChatContextItemsForMention(mentionQuery.text, token)
    if (!results?.length) {
        return null
    }
    const resultsAsMenuItems = results.map(result => ({
        key: getLabelForContextItem(result),
        item: result,
        shortLabel:
            result.type === 'symbol'
                ? `${result.kind === 'class' ? '$(symbol-structure)' : '$(symbol-method)'} ${
                      result.symbolName
                  }`
                : undefined,
    }))
    return resultsAsMenuItems
}
```

This adds support for URI as context:
![image](https://github.com/sourcegraph/cody/assets/68532117/6503c9b3-4c6b-40b1-9952-ecc656e3e250)